meetings/2024-01-17.md

```diff
@@ -0,0 +1,161 @@
+* MdJ: Want to make a suggestion into the spec of that wording?
+* eP: What do we think about instead of going through content types, like RDF and non-RDF sources. Then we don't have to list all patches.
+* SC: I asked for clarification in the chat. Some of that still applies. The requirement is more of a category of formats. The way it's written is not tied to a particular patch format. The only thing the spec says is patch is done this way, completely orthogonal to the formats that a server may be accepting, and if it does it needs to make available in ??? Right now the spec implies a server should be able to take an N3 document when you make a patch. Right now we're only saying RDF document but that is not entirely true. IF we want to say SPARQL is needed, that can be captured along those lines. There is nothing more generic to capture... right now we capture those two. 
+* MdJ: What about ??? not say anything about the body but the target. 
+* SC: Creating.
+* eP: in the context of creating. If you use SPARLQ Update patch to create an RDF resource, ???
+* MdJ: Continue in PR?
+* eP: It should covers 
+* SC: What notion do you know that would cover RDF, N3, SPARQL.. 
+* MdJ: Don;'t refer to body but target.
+* SC: You're saying if I give you an image you can give that back in turtle. 
+* MdJ: Let's continue on PR. 
+
+#### Add document status to guides, templates
+URL: https://github.com/solid/specification/pull/613
+
+Proposed by: SC
+
+* MdJ: You're marking some documents as no longer in effect..
+* SC: We're using w3c solid cg repo and that's the source of truth. Minor. 
+* VB: Approved. 
+
+#### Should Solid (storage) servers support "RDF documents" containing multiple subjects (or quads)?
+URL: https://github.com/solid/specification/issues/610
+
+Proposed by: Maxime
+
+* MdJ: This is more a question than a PR maybe?
+* ML: Some implementation servers are not allowing multiple subjects in the payload when creating or updating resources. So, can't in some apps to apply the TypeIndex or any self-describing document. Was wondering it should not be part of the spec if / when we are accepting RDF document / graph we must accept the graph contains multiple subjects.
+* MdJ: The spec allows the server. So can never be sure. If a server only accepts triples about the document, I'd say that server is broken.
+* eP: Besides self-describing docs, certain predicates are used in a certain direction, like JSON-LD reverse. This is common in RDF. Since it is a graph, often things can be subject or object. I think it'd be interetsing to add a test case. What level would this requirement would be.
+* MdJ: Even if you do WAC for instance, you'd have different Authorization rules. A test would fail in that case. That server wouldn't be compliant.
```

Comment:
```suggestion
* eP: Besides self-describing docs, certain predicates are used in a certain direction, like JSON-LD reverse. This is common in RDF. Since it is a graph, often things can be subject or object. I think it'd be interesting to add a test case. What level would this requirement be?
* MdJ: Even if you do WAC for instance, you'd have different Authorization rules. A test would fail in that case. That server wouldn't be compliant.
```